lib/license_finder/project_finder.rb

```diff
@@ -41,7 +42,7 @@ def project_root?(pathname)
 
     def active_project?(project_path)
       active_project = @package_managers.map do |pm|
-        pm.new(project_path: project_path).active?
+        pm.new(project_path: project_path).active? unless pm == GoWorkspace && @configurable
```

Comment:
Why the break in polymorphism? That is, why not pass `@configurable` to `pm.new`, and let GoWorkspace do something with it?

A related problem is that the name `@configurable` doesn't imply what it will be used for. Instead, maybe you could add an argument (possibly exposed at the CLI) which says which types of project managers to skip.